src/Stratis.Bitcoin/Utilities/ModelStateErrors/ModelStateErrors.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Stratis.Bitcoin.Utilities.JsonErrors;
+
+namespace Stratis.Bitcoin.Utilities.ModelStateErrors
+{
+    public static class ModelStateErrors
+    {
+        /// <summary>
+        /// Builds an <see cref="IActionResult"/> containing errors contained in the <see cref="ControllerBase.ModelState"/>.
+        /// </summary>
+        /// <returns>A result containing the errors.</returns>
+        public static IActionResult BuildErrorResponse(ModelStateDictionary modelState)
```

Comment:
I think this belongs to the Api feature because it will only ever be used when the feature is present.

Reply:
ah no because the Stratis.Bitcoin project also has controller where this is used.
Please ignore.